src/Codeception/Module/Phalcon.php

```diff
@@ -379,6 +379,29 @@ public function seeRecord($model, $attributes = [])
         $this->debugSection($model, json_encode($record));
     }
 
+    /**
+     * Checks that number of records exists in database.
+     *
+     * ``` php
+     * <?php
+     * $I->seeNumberOfRecords('App\Models\Categories', 3, ['name' => 'Testing']);
+     * ?>
+     * ```
+     *
+     * @param string $model Model name
+     * @param int $number int number of records
+     * @param array  $attributes Model attributes
+     * @part orm
+     */
+    public function seeNumberOfRecords($model, $number, $attributes = [])
+    {
+        $records = $this->findRecords($model, $attributes);
+        if ($records->count() != $number) {
+            $this->fail("Couldn't find $number records of $model with " . json_encode($attributes) .". Found: ".$records->count()." records.");
```

Comment:
Could you please do it more neat, using `sprintf` or something like that. I prefer avoid too long lines

Reply:
Sure, done.